### DISCUSSION TITLE
I get an error when trying to disable the loopback interface

### DISCUSSION THREAD

**Discussion by Hunam6** [tauri-apps/tauri#1071] [Q&A] [answered]
When executing this command `CheckNetIsolation.exe LoopbackExempt -a -n="Microsoft.Win32WebViewHost_cw5n1h2txyewy"` I get an error message: `Error occurred 1753 - 6D9`.

I'm on Windows 10 64 bit and I have no idea how to fix it.

**Comment by tensor-programming** [comment, marked as answer]
I would look here: https://social.technet.microsoft.com/Forums/en-US/cf70acd2-6d4d-4657-8931-56b0cafd9147/checknetisolation-produce-quoterror-occurred-1753-6d9quot?forum=win10itpronetworking
